tools/convert/README.md

```diff
@@ -24,38 +27,48 @@ Optional arguments:
   --output filename     'stdout' or a path to a file to write JSON output
                         to. Default is the input filepath with "gfxr" replaced
                         by "jsonl".
+  --format <format>     JSON format to write.
+           json         Standard JSON format (indented)
+           jsonl        JSON lines format (every object in a single line)
+  --include-binaries    Dump binaries from Vulkan traces in a separate file with an unique name. The main JSON file
+                        will include a reference with the file name. The binary files are dumped in a subdirectory
+  --expand-flags        Print flags values from Vulkan traces with its correspondent symbolic representation. Otherwise,
+                        the flags are printed as hexadecimal value.
+  --file-per-frame      Creates a new file for every frame processed. Frame number is added as a suffix
+                        to the output file name.
   --no-debug-popup      Disable the 'Abort, Retry, Ignore' message box
                         displayed when abort() is called (Windows debug only).
 ```
 
 
-The JSON document on each line is designed to be parsed by tools such as simple
+The JSON document is designed to be parsed by tools such as simple
 Python scripts as well as being useful for inspecting by eye after pretty
 printing, for example by piping through a command-line tool such as
 [`jq`](https://stedolan.github.io/jq/).
 For these post-processing use cases, `gfxrecon-convert` can be used to stream
 from binary captures directly, without
 having to save the intermediate JSON files to storage.
-Because each JSON object is on its own line, line oriented tools such as
+With `"--format jsonl"`, because each JSON object is on its own line, line oriented tools such as
 grep, sed, head, and split can be applied ahead of JSON-aware ones which
 are heavier-weight to reduce their workload on large captures.
 
 
 ## JSON Structure
 
-The tool's output is an ordered list of strings, one per line, each line a valid
-JSON document (JSON Lines). Below are the first few lines from a capture of vkcube, truncated to 200 columns.
+The tool's output is an ordered list of JSON structures, one per line in `--format jsonl` mode, each line a valid
```

Comment:
Could you clarify this a hair?  Maybe separate out the tool's output in default `json` mode versus `jsonl` mode; maybe something like:
```
The tool's output in default `json` mode is an ordered list of JSON structures, or a single JSON structure per line in `--format jsonl` mode.  Each structure represents metadata, a metadata command, or a graphics API function or method call.  Below are the first few lines from a capture of vkcube with convert applied with `--format jsonl`, truncated to 200 columns.
```

Reply:
Are all the subsequent examples either an extract of one structure from the JSON list or a pretty-print of a line from JSONL?  Could you add that as a note?